packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -4,19 +4,24 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-alert-width: Specifies the width of the component.
+ * @prop --calcite-alert-background-color: Specifies the component's background color.
+ * @prop --calcite-alert-title-text-color: Specifies the component's title text color.
+ * @prop --calcite-alert-message-text-color: Specifies the component's message text color.
+ * @prop --calcite-alert-corner-radius: Specifies the component's border radius.
+ * @prop --calcite-alert-close-background-color-hover: Specifies the component's close button background color when hovered or focused.
+ * @prop --calcite-alert-close-background-color-pressed: Specifies the component's close button background color when active.
+ * @prop --calcite-alert-dismiss-progress-background-color: Specifies the component's dismiss progress background color.
```

Comment:
similar to close I think the solution here is to consolidate the design pattern across components... 
i.e.
```
--calcite-dismiss-progress-background-color
```